src/redux/actions/index.js

```diff
@@ -25,32 +27,20 @@ export function constructContactItemsWithToken(token) {
         grade: contact.school.grade,
         id: contact._id,
         facebook_id: contact.facebook_id,
+        channel_id: contact.channel_id,
         fullContact: contact
       }));
       // TODO: once backend is updated, remove this and the PROD if condition
-      const usersWithChannels = [{
-        id: '5c15446bbf35ae4057222222',
-        name: 'Johnny',
-        channelSid: 'CH51a77d1a54a34ef296446df376d1110f'
-      }, {
-        id: '5c15446bbf35ae4057111111',
-        name: 'Nancy',
-        channelSid: 'CH51a77d1a54a34ef296446df376d1110f'
-      }];
+
       if (CONFIG.ENV !== 'PROD') {
-        contactData = contactData
-          .filter(contact => usersWithChannels.map(({ id }) => id).includes(contact.id))
-          .map((contact) => {
-            // eslint-disable-next-line no-param-reassign
-            contact.channelSid = usersWithChannels.find(({ id }) => id === contact.id).channelSid;
-            return contact;
-          });
+        contactData = contactData.filter(contact => contact.channel_id !== 'test');
```

Comment:
Do we need this filtering?

Reply:
yes, I think, when we run the script on API side to populate the DB I am putting channel_id for only users " Nancy & Jackson" which have the FB id also as we discussed. so all remaining users have channel_id "test" so this check is not showing that users which belong to channel test mean Twillio Chat doesn't work for them so I am filtering out.
- we can use different approaches like filter out users that have empty channel id?
<img width="481" alt="Screenshot 2020-02-22 at 8 11 35 PM" src="https://user-images.githubusercontent.com/55958033/75095224-86aa3f80-55b4-11ea-9830-5fd7be163bea.png">

<img width="544" alt="Screenshot 2020-02-22 at 8 11 18 PM" src="https://user-images.githubusercontent.com/55958033/75095141-d9372c00-55b3-11ea-87a4-644b24d9cd4b.png">

Reply:
That makes sense, let's leave it like you have it.